README.md

```diff
@@ -50,10 +50,20 @@ Library that helps identifying if the environment can be used for development of
 	 * Describes warning returned from nativescript-doctor check.
 	 */
 	interface IWarning {
-		/** The warning. */
+		/** The warning.
```

Comment:
this should be on the next line according to [JSDocs specification](http://usejsdoc.org/about-getting-started.html)